boards/st/nucleo_u083rc/nucleo_u083rc.dts

```diff
@@ -95,12 +112,14 @@
 	clock-frequency = <I2C_BITRATE_FAST>;
 };
 
+/*
 &i2c2 {
 	pinctrl-0 = <&i2c2_scl_pa7 &i2c2_sda_pa6>;
 	pinctrl-names = "default";
 	status = "okay";
 	clock-frequency = <I2C_BITRATE_FAST>;
 };
+*/
```

Comment:
Not quite clean for the reader.
Let's remove purely the node, if there is no use today, or provide a statement on the reason why it is commented out.